app/services/message_service.rb

```diff
@@ -0,0 +1,67 @@
+class MessageService
```

Comment:
@andela-tfowotade Why is this class added again in this PR.  You should always try to update your branch if a change is already on another branch. Please update your PR and remove all codes repeated in the PRs I've merged